rskj-core/src/main/java/co/rsk/peg/ErpFederation.java

```diff
@@ -2,39 +2,53 @@
 
 import co.rsk.bitcoinj.core.BtcECKey;
 import co.rsk.bitcoinj.core.NetworkParameters;
+import co.rsk.bitcoinj.script.RedeemScriptParserFactory;
 import co.rsk.bitcoinj.script.Script;
 import co.rsk.bitcoinj.script.ScriptBuilder;
 import co.rsk.peg.utils.EcKeyUtils;
 import java.time.Instant;
+import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 import org.ethereum.config.blockchain.upgrades.ActivationConfig;
 import static co.rsk.peg.FederationCreationException.Reason.INVALID_CSV_VALUE;
 import static co.rsk.peg.FederationCreationException.Reason.NULL_OR_EMPTY_EMERGENCY_KEYS;
 
-public abstract class ErpFederation extends Federation {
+public class ErpFederation extends Federation {
     protected static final long MAX_CSV_VALUE = 65_535L; // 2^16 - 1, since bitcoin will interpret up to 16 bits as the CSV value
     protected final List<BtcECKey> erpPubKeys;
     protected final long activationDelay;
     protected final ActivationConfig.ForBlock activations;
     protected Script standardRedeemScript;
     protected Script standardP2SHScript;
 
+    protected ErpRedeemScriptBuilder erpRedeemScriptBuilder;
```

Comment:
```suggestion
    private ErpRedeemScriptBuilder erpRedeemScriptBuilder;
```
Any reason to make it protected?

Reply:
is used in other places!